assets-ph1-website/scripts/common.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+{
+  const openBtn = document.getElementById("js-header-btn");
+  const menu = document.getElementById("js-menu");
+  const main = document.getElementById("js-main");
+  const footerLogo = document.getElementById("js-footer-logo");
+  const footerLine = document.getElementsByClassName("footer_line");
+  const footerSmall = document.getElementById("js-footer-small");
+  let index = 0;
+
+  openBtn.addEventListener("click", () => {
+    if (index === 0) {
```

Comment:
これopen closeに関わっているなら　
```suggestion
    if (open ===true) {
```

みたいになるようにするとわかりやすい！